### DISCUSSION TITLE
数据详细删除

### DISCUSSION THREAD

**Discussion by G0O96** [Planshit/Tai#208] [Ideas]
希望可以支持在特定时间和日期删除统计的网站和软件使用情况

**Comment by MC-dusk** [comment]
我就觉得很奇怪，右键一个统计项竟然不能删除

**Reply by MC-dusk** [reply]
希望增加几个选项，比如删除某个应用的所有记录，删除某个网站的所有记录，删除某天甚至某小时的所有记录，当然，最重要的，删除任意一个单独的记录
其实如果懒得做删除的交互，也可以导出数据，在外面处理完，再支持导入
